virt/getting_started/virt-using-the-cli-tools.adoc

```diff
@@ -28,3 +28,7 @@ include::modules/virt-deploying-libguestfs-with-virtctl.adoc[leveloffset=+1]
 
 include::modules/virt-about-libguestfs-tools-virtctl-guestfs.adoc[leveloffset=+2]
 
+[id="using_ansible"]
```

Comment:
```suggestion
[id="using_ansible__{context}"]
```
See [Anchoring in assembly files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-assembly-files).